cypress/support/resource.ts

```diff
@@ -13,20 +13,20 @@ Cypress.Commands.add('deleteResource', (kind: string, name: string, namespace?:
   // If cluster resource, ommit namespace
   if (!namespace) {
     cy.exec(
-      `kubectl delete --ignore-not-found=true --cascade ${kind} ${name} --wait=true --timeout=180s || true`,
-      { timeout: 180000 },
+      `oc delete --ignore-not-found=true --cascade ${kind} ${name} --wait=false --timeout=180s`,
```

Comment:
Does timeout meaningful here if `--wait==false`?

Reply:
Probably not, but this is what we have in downstream: https://gitlab.cee.redhat.com/cnv-qe/kubevirt-ui/-/blob/main/cypress/support/commands.ts#L52

Reply:
1. `--wait=false --timeout=180s` if wait is `false`, the timeout `180s` is no meaningful anymore, I would suggest to keep them as `true`.
2. keep the file as `commands.ts`

Reply:
changing to --wait=true